src/Providers/AuthUserProvider.tsx

```diff
@@ -42,26 +42,43 @@ export default function AuthUserProvider({
     enabled: !!localStorage.getItem(LocalStorageKeys.accessToken),
   });
 
+  const refresh = localStorage.getItem(LocalStorageKeys.refreshToken);
```

Comment:
use clear variable names. `refresh` is ambiguous. could be confused with other refresh's in this context.
```suggestion
  const refreshToken = localStorage.getItem(LocalStorageKeys.refreshToken);
```